docs/compose/BottomSheet/README.md

```diff
@@ -65,6 +66,7 @@ if (openBottomSheet) {
     BpkModalBottomSheet(
         state = state,
         content = { /* content of the bottom sheet */ },
+        dragHandleStyle = BpkDragHandleStyle.OnImage(),
```

Comment:
same here - can you add a separate example where you also set the type and describe what the difference in behaviour is

Reply:
Sure!